components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/SharedHttpClientConnectionManager.java

```diff
@@ -59,7 +61,9 @@ public SharedHttpClientConnectionManager(
       final List<SSLContextSelector> sslContextSelectors,
       @Named("${nexus.httpclient.connectionpool.size:-20}") final int connectionPoolSize,
       @Named("${nexus.httpclient.connectionpool.maxSize:-200}") final int connectionPoolMaxSize,
-      @Named("${nexus.httpclient.connectionpool.idleTime:-30s}") final Time connectionPoolIdleTime)
+      @Named("${nexus.httpclient.connectionpool.idleTime:-30s}") final Time connectionPoolIdleTime,
+      @Named("${nexus.httpclient.connectionpool.evictingDelayTime:-5s}") final Time connectionPoolEvictingDelayTime,
+      @Named("${nexus.httpclient.connectionpool.validateAfterInactivityTime:-2s}") final Time connectionPoolValidateAfterInactivityTime)
```

Comment:
just to confirm: the change to the default `validateAfterInactivity` time from -1 to 2s is expected?